tests/test_bar_data.py

```diff
@@ -204,6 +204,18 @@ def init_class_fixtures(cls):
 
         cls.ASSETS = [cls.ASSET1, cls.ASSET2]
 
+    def test_current_session(self):
+        for minute in self.trading_calendar.minutes_for_sessions_in_range(
```

Comment:
We should verify the behavior outside of the session as well for BTS.  I think the right behavior there is to roll forward.